src/test/scala/com/microsoft/hyperspace/TestUtils.scala

```diff
@@ -24,4 +27,11 @@ object TestUtils {
     result.state = state
     result
   }
+
+  def makeAbsolute(location: String): Path = makeAbsolute(new Path(location))
+
+  def makeAbsolute(path: Path): Path = {
```

Comment:
Can we create `PathUtils` so that it can be used in `factories.scala` as well?

Reply:
thanks @imback82, done